src/compass/s1_geocode_slc.py

```diff
@@ -197,6 +199,25 @@ def run(cfg: GeoRunConfig):
                                        weather_model_path=cfg.weather_model_file,
                                        delay_type=cfg.tropo_params.delay_type)
 
+            # If needed, make browse image and compute CSLC raster stats
+            browse_params = cfg.browse_image_params
+            if browse_params.enabled:
+                make_browse_image(out_paths.browse_path, output_hdf5,
+                                  cfg.bursts, browse_params.complex_to_real,
+                                  browse_params.percent_lo,
+                                  browse_params.percent_hi,
```

Comment:
```suggestion
                                  browse_params.percent_low,
                                  browse_params.percent_high,
```
Same opinion as @vbrancat in the comment above: https://github.com/opera-adt/COMPASS/pull/101/files#r1141075704